src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -154,6 +154,8 @@ void CascadiaSettings::_CreateDefaultProfiles()
     _profiles.emplace_back(cmdProfile);
 
     _globals.SetDefaultProfile(powershellProfile.GetGuid());
+
+    _globals.SetRequestedTheme(winrt::Windows::UI::Xaml::ElementTheme::Default);

```

Comment:
`CreateDefaultProfiles` isn't a great place for this .. is there something more generic or less default-profile-related to put it?